packages/hooks/src/__tests__/useAuthorize.spec.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { APIProvider, useFetch } from '@deriv/api';
+import { renderHook } from '@testing-library/react-hooks';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import useAuthorize from '../useAuthorize';
+
+jest.mock('@deriv/api', () => ({
+    ...jest.requireActual('@deriv/api'),
+    useFetch: jest.fn(),
+}));
+
+const mockUseFetch = useFetch as jest.MockedFunction<typeof useFetch<'authorize'>>;
+
+describe('useAuthorize', () => {
+    test('should return wallets list for the current loginid', () => {
```

Comment:
@aizad-deriv The test case for this hook can be improved, We should test to make sure the request will only be sent when a token exists, No need to test the response content 🙇🏻 

We can do it in the next PR 🙇🏻

Reply:
Noted @farzin-deriv thanks for the suggestions 🙇🏼‍♂️